.circleci/config.yml

```diff
@@ -63,7 +63,7 @@ jobs:
       - run:
           name: "Ensure we don't need to run 'make gen'"
           command: |
-            make gen && git diff --exit-code
+            make gen && go mod tidy && git diff --exit-code
```

Comment:
I don't understand why go.sum was changing in between but it was and causing failures on ci